### PR TITLE
motionblur: using MeshTools::compile for configuring the mesh

### PR DESCRIPTION
**Pull request by codewing**

### CODE REVIEW

File: src/motionblur/MotionBlurExample.cpp [hunk 1]
```diff
@@ -31,13 +31,12 @@
...
 #include <Magnum/GL/DefaultFramebuffer.h>
 #include <Magnum/GL/Renderer.h>
-#include <Magnum/MeshTools/CompressIndices.h>
-#include <Magnum/MeshTools/Interleave.h>
 #include <Magnum/Platform/Sdl2Application.h>
 #include <Magnum/Primitives/Icosphere.h>
 #include <Magnum/SceneGraph/Scene.h>
 #include <Magnum/Shaders/Phong.h>
 #include <Magnum/Trade/MeshData3D.h>
+#include <Magnum/MeshTools/Compile.h>
```
Comment (mosra) [inline src/motionblur/MotionBlurExample.cpp]: Order the includes alphabetically, please :)

File: CMakeLists.txt [hunk 2]
```diff
@@ -44,11 +44,11 @@ option(WITH_AREALIGHTS_EXAMPLE "Build Area Lights example (requires the Ui libra
 option(WITH_AUDIO_EXAMPLE "Build Audio example (requires the Audio library and the StbVorbisAudioImporter plugin)" OFF)
 option(WITH_BOX2D_EXAMPLE "Build Box2D integration example" OFF)
 option(WITH_BULLET_EXAMPLE "Build Bullet integration example (requires the BulletIntegration library)" OFF)
-cmake_dependent_option(WITH_CUBEMAP_EXAMPLE "Build CubeMap example (requires some JPEG importer plugin)" OFF "NOT MAGNUM_TARGET_GLES" OFF)
+cmake_dependent_option(WITH_CUBEMAP_EXAMPLE "Build CubeMap example (requires some JPEG importer plugin)" ON "NOT MAGNUM_TARGET_GLES" OFF)
 option(WITH_IMGUI_EXAMPLE "Build ImGui example" OFF)
 option(WITH_LEAPMOTION_EXAMPLE "Build LeapMotion example" OFF)
 option(WITH_MOUSEINTERACTION_EXAMPLE "Build Mouse Interaction example" OFF)
-cmake_dependent_option(WITH_MOTIONBLUR_EXAMPLE "Build Motion Blur example" OFF "NOT MAGNUM_TARGET_GLES" OFF)
+cmake_dependent_option(WITH_MOTIONBLUR_EXAMPLE "Build Motion Blur example" ON "NOT MAGNUM_TARGET_GLES" OFF)
```
Comment (mosra) [inline CMakeLists.txt]: Accidental change? :)

Comment (codewing) [reply CMakeLists.txt]: Oh :D

**Comment by mosra** [comment]
Merged as 092fb8aed5b3ad8c0e03410659467786baf24589. Thanks a lot!

**Pull request closed without merging**